src/lib/crypto/kimchi_bindings/js/bindings/oracles.js

```diff
@@ -75,31 +73,3 @@ function caml_pasta_fp_poseidon_params_create() {
 function caml_pasta_fq_poseidon_params_create() {
   return [0];
 }
-
-// Provides: caml_pasta_fp_poseidon_block_cipher
```

Comment:
we actually "remove" the old wasm functions so that we can overwrite them in `native/native-overrides` - once finished, we can go back to one file for both implementations but for now its easier to do it that way (because i couldnt think of an easier way to use the native conversion core for specific functions only - maybe there's some jsoo feature that would allow that but meh)